featurebyte/service/historical_feature_table.py

```diff
@@ -106,6 +112,20 @@ async def get_historical_feature_table_task_payload(
         else:
             observation_set_storage_path = None
 
+        # check number of features whether exceeds the limit
+        # if the task is triggered by a feature list, this limit is not applied
```

Comment:
Is the comment referring to a saved feature list?

Reply:
For this payload, when reaching this point it is either feature clusters or feature list is set. The comment here is referring to the case when feature list is set (which implies a saved feature list).